src/modules/esl-scrollbar/core/esl-scrollbar.ts

```diff
@@ -7,6 +7,7 @@ import {isMouseEvent, isTouchEvent, getTouchPoint, getOffsetPoint} from '../../e
 import {isRelativeNode} from '../../esl-utils/dom/traversing';
 import {TraversingQuery} from '../../esl-traversing-query/core';
 import {RTLUtils} from '../../esl-utils/dom/rtl';
+import {ScrollUtils} from '../../all';
```

Comment:
Please do not use massive cumulative bundles in the bounds of the library.
We can not guarantee that the final consumers will have tree shaking optimization working.